app/models/audio_file_template.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+
+class AudioFileTemplate < BaseModel
+  belongs_to :audio_version_template
+  acts_as_list scope: :audio_version_template
+
+  before_validation :set_defaults, on: :create
+
+  validates :length_minimum,
+            presence: true,
+            numericality: { less_than_or_equal_to: :length_maximum }
+
+  validates :length_maximum,
+            presence: true,
+            numericality: { greater_than_or_equal_to: :length_minimum }
+
+  def set_defaults
+    self.label ||= 'segment'
```

Comment:
Yeah, not deeply invested in this - the code in V3 is getting over the "not null" constraint by setting all labels to empty string. That drives me crazy, so I decided to give a bland value that could be overridden with something more colorful like "billboard" or "segment a" later on.

Do you have a better suggestion for a default?